.github/workflows/gradle.yml

```diff
@@ -10,48 +10,58 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Set up JDK
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v4
         with:
           distribution: temurin
           java-version: 17
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+        with:
+          # Publish a Gradle build scan so you can see all the build logs,
+          # a complete task timeline, test outputs, and the resolved dependencies of your build.
+          build-scan-publish: true
+          build-scan-terms-of-use-url: "https://gradle.com/terms-of-service"
+          build-scan-terms-of-use-agree: "yes"
       - name: Run Build
         id: build
-        uses: gradle/gradle-build-action@v2
+        run: ./gradlew build
         env:
           DEVELOCITY_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
```

Comment:
`DEVELOCITY_ACCESS_KEY` is not necessary here. A short-lived access key It is provided by `setup-gradle` thanks to `with: develocity-access-key: ...`